common-tasks/dispvm.md

```diff
@@ -11,74 +11,78 @@ redirect_from:
+In an AppVM's file manager, right click on the file you wish to open in a Disposable VM, then choose "Open in Disposable VM". 
+Wait a few seconds and the default application for this file type should appear displaying the file content. 
+This app is running in its own dedicated VM -- a Disposable VM created for the purpose of viewing or editing this very file. 
+Once you close the viewing application the whole Disposable VM will be destroyed. 
+If you have edited the file and saved the changes, the changed file will be saved back to the original AppVM, overwriting the original.
 
 ![r1-open-in-dispvm-1.png](/attachment/wiki/DisposableVms/r1-open-in-dispvm-1.png) ![r1-open-in-dispvm-2.png](/attachment/wiki/DisposableVms/r1-open-in-dispvm-2.png)
 
 Opening a fresh web browser instance in a new Disposable VM
 -----------------------------------------------------------
 
-Sometimes it is convenient to open a fresh instance of Firefox within a new fresh Disposable VM. This can be easily done by using the Start Menu: just go to Start -\> System Tools -\> DispVM:Firefox web browser . Wait a few seconds until a web browser starts. Once you close the viewing application the whole Disposable VM will get destroyed.
+Sometimes it is desirable to open an instance of Firefox within a new fresh Disposable VM. 
+This can be done easily using the Start Menu: just go to Start -\> System Tools -\> DispVM:Firefox web browser. 
+Wait a few seconds until a web browser starts. 
+Once you close the viewing application the whole Disposable VM will get destroyed. 
 
 ![r1-open-in-dispvm-3.png](/attachment/wiki/DisposableVms/r1-open-in-dispvm-3.png)
 
 Opening a file in a Disposable VM via command line (from AppVM)
 ---------------------------------------------------------------
 
-Use the `qvm-open-in-dvm` command line (from your AppVM), e.g.:
+Use the `qvm-open-in-dvm` command from your AppVM:
 
 ~~~
 [user@work-pub ~]$ qvm-open-in-dvm Downloads/apple-sandbox.pdf
 ~~~
 
-The qvm-open-in-dvm will not exit until you close the application in the Disposable VM.
-
-Starting an arbitrary application in a disposable VM via command line (from Dom0)
----------------------------------------------------------------------------------
-
-**Note:** Normally there should be no need for doing this -- this is just for Qubes hackers ;)
-
-~~~
-[joanna@dom0 ~]$ echo xterm | /usr/lib/qubes/qfile-daemon-dvm qubes.VMShell dom0 DEFAULT red
```

Comment:
We get questions about this all the time.

Reply:
Okay! I'll clean this section up.